tests/bdd/scale/test_NAS_T1123.py

```diff
@@ -0,0 +1,142 @@
+        driver.find_element_by_xpath('//input[@data-placeholder="Username"]').clear()
+        driver.find_element_by_xpath('//input[@data-placeholder="Username"]').send_keys('root')
+        driver.find_element_by_xpath('//input[@data-placeholder="Password"]').clear()
+        driver.find_element_by_xpath('//input[@data-placeholder="Password"]').send_keys(root_password)
+        assert wait_on_element(driver, 5, '//button[@name="signin_button"]')
+        driver.find_element_by_xpath('//button[@name="signin_button"]').click()
+    else:
+        driver.find_element_by_xpath('//mat-list-item[@ix-auto="option__Dashboard"]').click()
+
+
+@when('you should be on the dashboard, click on Sharing then Windows Shares(SMB)')
+def you_should_be_on_the_dashboard_click_on_sharing_then_windows_sharessmb(driver):
+    """you should be on the dashboard, click on Sharing then Windows Shares(SMB)."""
+    time.sleep(1)
+    assert wait_on_element(driver, 10, '//mat-list-item[@ix-auto="option__Dashboard"]', 'clickable')
+    driver.find_element_by_xpath('//mat-list-item[@ix-auto="option__Dashboard"]').click()
+    time.sleep(1)
+    assert wait_on_element(driver, 10, '//span[contains(.,"Dashboard")]')
+    driver.find_element_by_xpath('//mat-list-item[@ix-auto="option__Shares"]').click()
+
+
+@then('The Windows Shares(SMB) page should open, Click Add')
+def the_windows_sharessmb_page_should_open_click_add(driver):
+    """The Windows Shares(SMB) page should open, Click Add."""
+    time.sleep(1)
+    assert wait_on_element(driver, 5, '//mat-card[contains(.,"SMB")]//button[@ix-auto="button__-add"]')
+    driver.find_element_by_xpath('//mat-card[contains(.,"SMB")]//button[@ix-auto="button__-add"]').click()   
+
+
+@then(parsers.parse('Set Path to the ACL dataset "{path}", Input "{smbname}" as name, Click to enable, Input "{description}" as description, and Click Summit'))
+def set_path_to_the_acl_dataset_mntsystemkmy_acl_dataset_input_mysmbshare_as_name_click_to_enable_input_my_smb_test_share_as_description_and_click_summit(driver, path, smbname, description):
+    """Set Path to the ACL dataset "{path}", Input "{smbname}" as name, Click to enable, Input "{description}" as description, and Click Summit."""
+    time.sleep(1)
+    global smb_path
+    smb_path = path
+    """Set Path to the ACL dataset "/mnt/system/my_acl_dataset"."""
+    assert wait_on_element(driver, 5, '//input[@ix-auto="input__path"]')
+    driver.find_element_by_xpath('//input[@ix-auto="input__path"]').clear()
+    driver.find_element_by_xpath('//input[@ix-auto="input__path"]').send_keys(path)
+    time.sleep(1)
+    assert wait_on_element(driver, 5, '//input[@ix-auto="input__Name"]')
+    driver.find_element_by_xpath('//input[@ix-auto="input__Name"]').clear()
+    driver.find_element_by_xpath('//input[@ix-auto="input__Name"]').send_keys(smbname)
+    checkbox_checked = attribute_value_exist(driver, '//mat-checkbox[@ix-auto="checkbox__Enabled"]', 'class', 'mat-checkbox-checked')
```

Comment:
optional but for caution I would add a wait on '//mat-checkbox[@ix-auto="checkbox__Enabled"]' to be clickable.